Backend/app/rider-platform/rider-app-drainer/server/Main.hs

```diff
@@ -79,3 +82,21 @@ getConnectionString dbConfig =
       <> dbConfig.connectPassword
       <> " port="
       <> show dbConfig.connectPort
+
+createPoolConfig :: EsqDBConfig -> IO (PoolConfig Connection)
+createPoolConfig dbConfig = do
```

Comment:
this function can be pure, there is no IO actions executed:
`mkPoolConfig :: EsqDBConfig -> PoolConfig Connection`